### ISSUE TITLE
My Personal Issues

### ISSUE THREAD

**Issue by GottaLoveAng** [khcrysalis/Feather#32]
1. I've seen that you respond to most issues, so I thought I should give my input! First off, I love Feather, and I'm currently using it as an alternative to Esign. However, one of my main issues with it is the fact that it seems to take up a lot of storage with no way to clear it. I've deleted imported apps, and checked the Files app for any residual files. It just seems to be rapidly increasing with no way to clear it completely. I've had the same issue with Esign as well, but hopefully there's a fix for this in the works? Or a fix that I'm unaware of? (For context, I pressed share on a 2gb IPA, and it suddenly added 2gb to the app with no way to clear it. Simply signing an IPA creates unclearable storage as well.)
![image](https://github.com/user-attachments/assets/e8767b51-ca44-4392-8898-415f30294e20)

![image](https://github.com/user-attachments/assets/8b174bfc-d8b0-41c0-9fc8-462eaa90059e)

![image](https://github.com/user-attachments/assets/cfb565e6-33f3-4012-a7a2-38b7f9374d41)

2. Would it be possible to include a sort of like, "installation console?" Similarly to Esign's installation process. Or perhaps, some sort of progress bar? It feels a bit weird to just sit there without seeing what's happening besides vague text in my opinion.

3. I've never seen this implemented in other signing apps, so I'm unsure if it's possible, but would it be possible to install an IPA from a link, rather than importing it? Or perhaps signing the IPA file directly rather than importing it? Importing uses much more storage than needed. (Which seems like it's not possible to completely clear at the moment.) 

4. Could you possibly add Feather to the share sheet? It makes it a lot easier to just send IPAs directly to the app similarly to selecting esign from the share sheet. Or possibly making Feather the default app to open IPAs in files, similarly to "Scarlet?" (Bad app, I know. I only _occasionally_ use it with my dev certs though.) Although, these are sort of useful, but niche features. So, I understand why you may not add this functionality.

I appreciate you taking your time to read this, and although Feather is in beta, and I'm aware a lot of things are still being worked on, it's already an amazing app! That's all though, thank you!

**Comment by khcrysalis** [comment]
> Which seems like it's not possible to completely clear at the moment

It's possible, rather all imported files are in the apps `/tmp` directory which gets cleared every relaunch of the app

When signing, the app does get duplicated and put into the signing directory.

I've made sure there were minimal storage problems, which I have accomplished but your situation is very odd. 

I would recommend you wait on iOS to fully recognize you deleted the files because it kay not recognize immediately

**Comment by khcrysalis** [comment]
Also yes I plan to add it to the share sheet (i already have, just not released yet), just take a look at the roadmap

**Comment by GottaLoveAng** [comment]
Seems like the storage _did_ clear itself eventually. Good to know! Also, if you don't mind me asking, what's your opinion on implementing the 3rd suggestion? I've recently seen you respond to the 2nd one in another issue, and I didn't mean to ask again, sorry about that.

**Comment by khcrysalis** [comment]
> I've never seen this implemented in other signing apps, so I'm unsure if it's possible, but would it be possible to install an IPA from a link, rather than importing it? Or perhaps signing the IPA file directly rather than importing it? 

Most likely not, I tried to go with the idea that the user should be able to manage files themselves

**Comment by GottaLoveAng** [comment]
Okay, I understand. Thank you for responding!
